index.html

```diff
@@ -253,9 +253,22 @@ <h4>Skills</h4>
 
 
         </div>
+           <section id="contact-me">
+            <h3 class="contatct-head">Contact me</h3>
+            <p class="contact-desc">If you have an application you <br> are interested in developing, a <br> feature that you
+                need built or a <br> project that needs coding. <br>
+                I'd love to help with it!</p>
```

Comment:
- [ ] Kindly make sure the style of this text stick to the Figma design 😄 , the text-align, the color  of this text should be the same as in the Figma design 👍 

| Your design | Figma design |
| :---: | :---: |
| ![Capture 7](https://user-images.githubusercontent.com/80458199/230501732-1a332b6d-1837-441d-aa3f-5c3a8a7c1754.PNG) | ![Capture 5](https://user-images.githubusercontent.com/80458199/230501561-dc01a930-cc77-4e5c-a502-998dd2b937df.PNG) |